src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ServiceProvider.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+
+@Entity
+data class ServiceProvider(
+  @Id val id: UUID,
```

Comment:
there's an open question about the service provider id. it seems sensible we should map it to the hmpps auth 'groupCode' for the provider. in which case it would be a string data type. let's confirm when we talk through this in person.

Reply:
updated to string type (var char(30) in sql)